common/src/service/ExceptionHandlingHelper.cs

```diff
@@ -18,7 +19,7 @@ class ExceptionHandlingHelper
             var mappings = new Dictionary<HttpStatusCode, Func<HttpResponseMessage, Task<Exception>>>();
 
             mappings.Add(HttpStatusCode.NoContent, async (response) => new DeviceNotFoundException(await GetExceptionMessageAsync(response).ConfigureAwait(false)));
-            mappings.Add(HttpStatusCode.NotFound, async (response) => new DeviceNotFoundException(await GetExceptionMessageAsync(response).ConfigureAwait(false)));
+            mappings.Add(HttpStatusCode.NotFound, HttpNotFoundMapping);
```

Comment:
Trying to understand why is this done differently now?

Reply:
'DeviceNotFound' and 'DeviceNotOnline' both have 404 error code in http header, so there is only one mapping for both